nucypher/blockchain/eth/agents.py

```diff
@@ -356,69 +341,76 @@ def get_global_locked_tokens(self, at_period: int = None) -> int:
 
         Returns an amount of NuNits.
         """
-        if at_period is None:
+        if not at_period:
```

Comment:
Is having `at_period == 0` a possibility? Would this branch be the desired behavior then?

Reply:
Hmm, 0 is valid. Probably safer to keep it checking for `None`.

Reply:
:heavy_check_mark: